scripts/tests/test_validate_docstrings.py

```diff
@@ -594,7 +603,82 @@ def test_bad_generic_functions(self, func):
                      marks=pytest.mark.xfail)
     ])
     def test_bad_examples(self, capsys, klass, func, msgs):
-        validate_one(self._import_path(klass=klass, func=func))  # noqa:F821
-        err = capsys.readouterr().err
+        result = validate_one(self._import_path(klass=klass, func=func))  # noqa:F821
         for msg in msgs:
-            assert msg in err
+            assert msg in ' '.join(result['errors'])
+
+
+class TestApiItems(object):
```

Comment:
Also minor but since this test isn't used for any testing (i.e. not collected by the runner) can we remove the `Test` prefix?